src/test/java/io/github/nianna/TestUtil.java

```diff
@@ -38,4 +38,14 @@ public static List<String> loadPlPatterns() {
         }
     }
 
+    public static List<String> loadDaPatterns() {
+        try {
```

Comment:
Please extract private method called e.g. _loadPatterns(String sourceFile)_ to avoid copying code

Reply:
Sure. Pushed in 0979497